CHANGELOG.md

```diff
@@ -91,7 +94,7 @@
 * [ENHANCEMENT] Query-tee: added support to `/metadata`, `/alerts`, and `/rules` endpoints #2600
 * [ENHANCEMENT] Query-tee: added support to query results comparison between two different backends. The comparison is disabled by default and can be enabled via `-proxy.compare-responses=true`. #2611
 * [ENHANCEMENT] Query-tee: improved the query-tee to not wait all backend responses before sending back the response to the client. The query-tee now sends back to the client first successful response, while honoring the `-backend.preferred` option. #2702
-* [ENHANCEMENT] Thanos and Prometheus upgraded. #2604 #2634 #2686
+* [ENHANCEMENT] Thanos and Prometheus upgraded. #2602 #2604 #2634 #2659 #2686 #2756
   * TSDB now supports isolation of append and queries.
```

Comment:
This is technically true, but only on single TSDB. I'm not sure it makes sense to mention it for Cortex, since asking different ingesters may return inconsistent results. :(

Reply:
Removed.